lists/pancakeswap-arbitrum-default.json

```diff
@@ -181,6 +181,14 @@
       "decimals": 18,
       "logoURI": "https://tokens.pancakeswap.finance/images/arbitrum/0x0c880f6761F1af8d9Aa9C466984b80DAb9a8c9e8.png"
     },
+    {
+      "name": "mPendle",
+      "symbol": "mPENDLE",
```

Comment:
mPendle

Reply:
let's do `mPENDLE`. Last time we received complaint and request to switch from `mCake` to `mCAKE`